src/main/app/src/hooks.js

```diff
@@ -17,7 +17,9 @@ export const useLanguageState = () => {
   const location = useLocation();
   const history = useHistory();
   const lng = location.pathname.match(/^\/(.*?)(\/|$)/)?.[1];
-  document.documentElement.setAttribute('lang', lng || 'fi');
+  if (!!lng) {
```

Comment:
Jos on tarve muuntaa booliksi, niin käytetään mieluummin Boolean()-funktiota
```suggestion
  if (Boolean(lng)) {
```